3p/integration.js

```diff
@@ -180,6 +180,7 @@ import {my6sense} from '#ads/vendors/my6sense';
 import {myfinance} from '#ads/vendors/myfinance';
 import {myoffrz} from '#ads/vendors/myoffrz';
 import {mytarget} from '#ads/vendors/mytarget';
+import {myua} from '../ads/vendors/myua';
```

Comment:
We no longer need changes in this file. Sorry this likely changed while you were working on this pr please revert.